openstates/mt/__init__.py

```diff
@@ -82,3 +82,8 @@ def get_organizations(self):
         yield legislature
         yield upper
         yield lower
+
+    def get_session_list(self):
+        from openstates.utils.lxmlize import url_xpath
```

Comment:
No big deal, but I'm not sure why the import is happening inside the method. Any reason not to move to the module level?